engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/nLayout/area/impl/TextArea.java

```diff
@@ -115,12 +121,27 @@ public int getTextLength() {
 		return textLength;
 	}
 
-	private String calculateText() {
+	public String getRawText() {
```

Comment:
Does this really need to be public (API)

Always add proper javadoc for public methods